job_definitions/notify-suppliers-reminder-sign-agreement.yml

```diff
@@ -0,0 +1,73 @@
+- job:
+    name: "remind-suppliers-to-sign-framework-agreement-(43)"
+    display-name: "Remind suppliers to sign framework agreement (43)"
+    project-type: pipeline
+    description: "Send a reminder email to suppliers who have at least one successful lot entry on the given framework but who have not yet signed their framework agreement."
+    parameters:
+      - choice:
+          name: STAGE
+          choices:
+            - preview
+            - staging
+            - production
+      - string:
+          name: FRAMEWORK_SLUG
+          default: null
+          description: "Framework Slug e.g. 'digital-outcomes-and-specialists-5'"
+      - string:
+          name: SUPPLIER_IDS
+          default: null
+          description: "Comma-separated list of Supplier IDs to be emailed (e.g. '1234,5678')"
```

Comment:
Is it obvious to someone using this job what happens if this is left empty?

Reply:
Good point. I can update the description to make this clearer

Reply:
added in 38c3195